host/4/bullseye/amd64/java/java8-zulu/java8-composite.template

```diff
@@ -0,0 +1,34 @@
+ARG JAVA_VERSION=8u252
+
+FROM mcr.microsoft.com/java/jre-headless:${JAVA_VERSION}-zulu-debian10-with-tools as jre
```

Comment:
This one might also be on Buster not bullseye

Reply:
will update once java build image is push to mcr